lua/gitlab/reviewer/init.lua

```diff
@@ -1,42 +0,0 @@
--- This Module will pick the reviewer set in the user's
```

Comment:
I personally would maybe leave this here because it nicely abstracts reviewer, we have control over functions which we use elsewhere in code ( assuming we use only things exposed here ) and it will allow for future extensibility if someone else want to support other reviewers ( or build some kind of plugin ). 

What do you think ? ( if you feel like it is bad idea feel free to ignore this :+1: )

Reply:
Sure, will put it back. No problem, thanks!